pkgs/development/tools/rbspy/default.nix

```
@@ -43,6 +44,10 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ ruby which ];
 
+  passthru.updateScript = nix-update-script {
+    attrPath = pname;
+  };
```

```suggestion
  passthru.updateScript = nix-update-script { };
```

`attrPath` is no longer required.